app/private_publisher.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
```

Comment:
This is Python boilerplate and common practice. See
[Encoding declarations](https://docs.python.org/3/reference/lexical_analysis.html#encoding-declarations). It's not essential here and since this is the only file that has it I'll take it out.